src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExporterExtensions.cs

```diff
@@ -0,0 +1,69 @@
+// <copyright company="Dynatrace LLC">
+// Copyright 2021 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Metrics;
+
+namespace Dynatrace.OpenTelemetry.Exporter.Metrics
+{
+	public static class DynatraceMetricsExporterExtensions
+	{
+		/// <summary>
+		/// Adds <see cref="DynatraceMetricsExporter"/> to the <see cref="MeterProviderBuilder"/>.
+		/// </summary>
+		/// <remarks>
+		/// The exporter is configured together with a <see cref="PeriodicExportingMetricReader"/>.
+		/// By default, the export interval is 1 minute (60000ms). It can be customized via the
+		/// <see cref="DynatraceExporterOptions.MetricExportIntervalMilliseconds"/> property.
+		/// </remarks>
+		/// <param name="builder"><see cref="MeterProviderBuilder"/> builder to use.</param>
+		/// <param name="configure">Exporter configuration options.</param>
+		/// <param name="loggerFactory">The logger factory used to generate a ILogger instance for the exporter.</param>
+		/// <returns>The instance of <see cref="MeterProviderBuilder"/> to chain the calls.</returns>
+		public static MeterProviderBuilder AddDynatraceExporter(
+			this MeterProviderBuilder builder,
+			Action<DynatraceExporterOptions> configure = null,
```

Comment:
whats the point of passing an action here over simply passing the `DynatraceExporterOptions` object?

Reply:
It's I guess a combination of "style" (A lot of these `.Add` in .NET are following this style), but also the fact that it's "lazy" and you don't have to create the instance yourself.